dydx/dydxPresenters/dydxPresenters/_v4/Vault/DepositsAndWithdrawals/dydxVaultDepositWithdrawViewBuilder.swift

```diff
@@ -224,3 +229,15 @@ extension dydxViews.VaultTransferType {
         }
     }
 }
+
+extension VaultAccount {
+    var roundedWithdrawableUsdc: Double? {
+        withdrawableUsdc?.doubleValue.round(to: 2, rule: .towardZero)
```

Comment:
where does `withdrawableUsdc` come from?

old code uses balanceUsdc

`let roundedBalance = vault?.account?.balanceUsdc?.doubleValue.round(to: 2, rule: .towardZero) ?? 0`

Reply:
From backend field withdrawableEquity... Seems like the correct one to use.